force-app/main/default/classes/FlowControlLoopsCollections.cls

```diff
@@ -29,7 +29,12 @@ public with sharing class FlowControlLoopsCollections {
      * @return "Hello World!" if x greater than y, otherwise return null.
      */
     public static String helloWorld(Integer x, Integer y) {
-        return null; // Replace null with the variable you used to store the result
+        string str;
+        if(x < y || x == null || y == null || x == y){
+            return null;
+        }
+        str = 'Hello World!';
+        return str; // Replace null with the variable you used to store the result
```

Comment:
Really awesome work here, this is exactly how you should structure your method. However, instead of declaring a string, just return the value.